timetable/XML/XmlCreator.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Xml;
+using System.Xml.Linq;
+
+
+namespace Timetable.timetable.XML
+{
+	public class XmlCreator
```

Comment:
As stated earlier, we should look into the [XmlSerializer](https://docs.microsoft.com/en-us/dotnet/api/system.xml.serialization.xmlserializer?view=netframework-4.7.2) because that basically does exactly what we want to to.

Reply:
I disagree, using XElements and Xdocuments is easier to read in my opinion.  And this creator is simple and consists everything we need.